omegaconf/_utils.py

```diff
@@ -58,10 +101,10 @@ def no_duplicates_constructor(
             mapping[key] = value
         return loader.construct_mapping(node, deep)
 
-    class SafeLoaderWrapper(yaml.SafeLoader):  # type: ignore
+    class OmegaConfLoader(yaml.SafeLoader):  # type: ignore
         pass
 
-    loader = SafeLoaderWrapper
+    loader = OmegaConfLoader
     loader.add_implicit_resolver(
         "tag:yaml.org,2002:float",
         re.compile(
```

Comment:
this should happen in the constructor of OmegConfLoader.

Reply:
I am landing this because I am releasing another rc, but please follow up on this one in another diff.

OmegaConfLoader should work like OmegaConfDumper.